README.rst

```diff
@@ -58,4 +58,6 @@ Django version  Python versions
 ==============  ========================
 3.2 LTS         3.6, 3.7, 3.8, 3.9, 3.10
 4.0             3.8, 3.9, 3.10
+4.1             3.8, 3.9, 3.10, 3.11
+4.2             3.8, 3.9, 3.10, 3.11
```

Comment:
Please either remove this line (version 4.2), or add django 4.2 to the `tox.ini` and `setup.py` files as well.